.github/workflows/ci.yml

```diff
@@ -10,19 +10,15 @@ on:
   workflow_dispatch:
 
 jobs:
-  validation:
+  validation-8:
```

Comment:
This is 7.4+8.1 as outlined below and in the documentation

Reply:
There are also references in our other libs: https://github.com/spryker-eco/loggly/blob/master/.github/workflows/ci.yml